src/pages/errorPage/errorPage.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useDidMount } from '../../utils/hooks';
+import './errorPage.scss';
```

Comment:
import code block should be separated from the other part of the file by a space